lib/robot.js

```diff
@@ -4,6 +4,9 @@ const Context = require('./context')
 const logger = require('./logger')
 const GitHubApi = require('./github')
 
+// List of events causing GitHub App to be unable to authenticate itself (#382):
+const breakingEvents = ['installation.created', 'installation.deleted']
```

Comment:
`installation.created` will work fine (won't cause errors), but `installation` will also trigger when `installation.deleted` is triggered. This is because the `.something` looks inside the payload for an `action` field; but really, the event that GitHub sends is an `installation` event. You'll probably need to have a similar check, but within the actual event handler to check against the event _and_ action.